conf/nginx.conf

```
@@ -139,12 +139,15 @@ location ^~ __PATH__/ {
   # Serve static files
   location ~ \.(?:css|js|mjs|svg|gif|ico|jpg|png|webp|wasm|tflite|map|ogg|flac)$ {
     try_files $uri / __PATH__/index.php$request_uri;
-    expires 6M;         # Cache-Control policy borrowed from `.htaccess`
-    access_log off;     # Optional: Don't log access to assets
-
-    location ~ \.wasm$ {
-      default_type application/wasm;
-    }
+    # HTTP response headers borrowed from Nextcloud `.htaccess`
```

```suggestion
    # Adapted from https://github.com/nextcloud/documentation/blob/master/admin_manual/installation/nginx-root.conf.sample
    # HTTP response headers borrowed from Nextcloud `.htaccess`
```

(i assume that's where it's from ?)